include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -378,6 +414,16 @@ namespace parallel
        */
       Settings settings;
 
+      /**
+       * Partitioner used in copy_triangulation().
+       */
+      std::function<void(dealii::Triangulation<dim, spacedim> &,
+                         const unsigned int)>
+        partitioner = [](dealii::Triangulation<dim, spacedim> &tria,
+                         const unsigned int                    n_partitions) {
+          GridTools::partition_triangulation_zorder(n_partitions, tria);
```

Comment:
Initializers at class scope have some funny semantics, and we've generally avoided them. Just put a call to `set_partitioner()` into the constructor of the class with this lambda function.

Reply:
Or use a member initializer.